src/shared/securebits-util.h

```diff
@@ -7,7 +7,7 @@ int secure_bits_to_string_alloc(int i, char **s);
 int secure_bits_from_string(const char *s);
 
 static inline bool secure_bits_is_valid(int i) {
-        return ((SECURE_ALL_BITS | SECURE_ALL_LOCKS) & i) == i;
+        return FLAGS_SET(SECURE_ALL_BITS | SECURE_ALL_LOCKS, i);
```

Comment:
Technically, this is valid. But it's super misleading, because the flags are here in the position of the value. Maybe skip this one?